refinery/ui/source/styles/file-browser.less

```diff
@@ -267,8 +267,10 @@
   }
 }
 
-#node-selection-popover {
-  width: 400px;
+#assay-files-table {
```

Comment:
I think you can just do:
```
#assay-files-table.popover {
 width: 400px !important;
}
```

But thats just syntax preference, not sure what we prefer